content/articles/machine-learning-image-classification-with-javascript-and-nyckel/index.md

```diff
@@ -0,0 +1,328 @@
+
+- [ML model integration with the Web Application](#ml-model-integration-with-the-web-application)
+
+- [App live testing](#app-live-testing)
+
+- [Conclusion](#conclusion)
+
+
+### What is Nyckel?
+Nyckel is a Machine Learning API that provides automated training to models instead of manually writing the code. It also allows these models to be integrated into applications with the best simplicity.
+
+### Requirements
+To develop, you will need the following;
+- A suitable editor like visual studio, sublime and others.
+- Prior knowledge to Nyckel, css, Javascript.
+- HTML basics.
+
+With these requirements in place, the coding journey can kick off.
+Follow each step below to develop your own Machine Learning-enabled web application.
+
+### Javascript CSS and HTML website build
+We are going to build a simple web application where you can upload a photo from the desktop into the app. The layout will only need a section to place your photo and an button to make a system call to the file explorer.
+
+#### **HTML Build-up**
+Below are the steps involved in HTML build-up. Create a file with .html extension and follow the following steps.
+This code is placed under the head tag creates a connection to the Javascript file.
+
+```html
+<html>
+
+<head>
+  <script src="https://code.jquery.com/jquery-latest.min.js"></script>
+</head>
+
+```
+Under the body tag is where the upload button and the upload section lies. This code creates what is required under the body section without any styling.
+
+```html
+<body>
+
+  <div id="main" class="container">
+
+    <h1 id="title">Mask or no mask?</h1>
+
+    <!-- Drag and Drop container-->
+    <div class="upload-area" id="uploadfile">
+      <h2>Drop Your photo here</h2>
+    </div>
+    <input type="file" name="file" id="file">
+    <a id="nyckel" href="https://www.nyckel.com">Machine Learning application created by Victor</a>
+  </div>
+</body>
+```
+When you run the combination of the code above, you should have a web page with a button and an upload section only as shown in the photo below.
+
+![htmlResult](/engineering-education/machine-learning-image-classification-with-javascript-and-nyckel/htmlResult.png)
+
+The photo above shows the results with no styling applied. Since this tutorial is not deeply involved in website styling, you can apply your own styles
+and learn how to link them to the application. For more information on css styling, you can refer to this [article](https://www.section.io/engineering-education/getting-started-with-css/) for CSS beginners.. To get my styles that have been used for this particular tutorial, you can refer to my github [repository](https://github.com/del-ui/Machine-Learning-Image-Classifier-Application/blob/main/style.css) to get the full CSS code.
+
+When you use the code from the above link, similar results as those in the photo below are obtained.
+![styled](engineering-education/machine-learning-image-classification-with-javascript-and-nyckel/styled.png)

```

Comment:
Fix this image path, there are multiple occurrence with this minor omission of '/' at the start!